### ISSUE TITLE
'selected' attribute is not working on 'select' type fields on v2 Forms

### ISSUE THREAD

**Issue by gtaranti** [petalframework/petal_components#197]
It used to work on v1 Forms.
When 'selected' attribute was used, the option was selected on page load.
But on v2 Forms setting the 'selected' attribute does not work. The select element always uses the 1st option.

```
@durations [
    {"5 minutes", 5 * 60},
    {"7 minutes", 7 * 60},
    {"15 minutes", 15 * 60},
    {"30 minutes", 30 * 60},
    {"45 minutes", 45 * 60},
    {"1 hour", 60 * 60}
]

<.field
     type="select"
    class="w-max"
    options={@durations}
     field={@form[:duration]}
     label="Total duration"
     selected={@auction.duration || "3600"}
/>
```

**Comment by mplatts** [comment]
Should be fixed in v1.2.6

**Comment by gtaranti** [comment]
Thanks!
